core/template.yaml

```diff
@@ -200,6 +200,42 @@ Resources:
       AliasName: !Sub alias/${AWS::StackName}/${Environment}/database-kms-key
       TargetKeyId: !Ref DatabaseKmsKey
 
+  EncryptionGeneratorKmsKey:
```

Comment:
This looks to have too specific a role for the core stack. Maybe it should just go in audit?

Reply:
The point of this stack is that it is things that we will need in every account. So like generic KMS keys, an S3 Logs bucket etc.